### PR TITLE
Add tcp sink

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

**Review by ced0ps — APPROVED** [review]
LGTM if tested

File: helm/fluent-logshipping-app/.kube-linter.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+checks:
+  addAllBuiltIn: true
+  exclude:
+    - *
```
Comment (ced0ps) [inline helm/fluent-logshipping-app/.kube-linter.yaml:4]: please consider actually listing the tests you need to skip. The linter is there for a reason.

File: helm/fluent-logshipping-app/values.yaml [hunk 2]
```diff
@@ -101,6 +101,21 @@ outputs:
...
+    host: ""
+    port: ""
+    # Specify the data format to be printed. Supported formats are msgpack, json, json_lines and json_stream.
+    format: json
+    # Specify the name of the time key in the output record. To disable the time key just set the value to false.
+    json_date_key: date
+    # Specify the format of the date. Supported formats are double, epoch and iso8601 (eg: 2018-05-30T09:39:52.000681Z)
+    json_date_format: double
+    inputLogTypes: *logTypes
+    indices:
```
Comment (ced0ps) [inline helm/fluent-logshipping-app/values.yaml:115]:
```suggestion
    # This output currently only supports the forwarding of syslog messages.
    indices:
```

File: helm/fluent-logshipping-app/.kube-linter.yaml (hunk 1 above)
Comment (T-Kukawka) [reply helm/fluent-logshipping-app/.kube-linter.yaml]: done

**Pull request merged**